example/lib/main.dart

```diff
@@ -92,6 +92,10 @@ class _ChatScreenState extends State<ChatScreen> {
           backgroundColor: theme.backgroundColor,
         ),
         sendMessageConfig: SendMessageConfiguration(
+          imagePickerIconsConfig: ImagePickerIconsConfiguration(
+            cameraIconColor: theme.cameraIconColor,
```

Comment:
why are we giving only control of color and not add user decided icon/image.

Reply:
We have given two properties for custom icon/image with Widget type, But in this example for theme we have added only color configuration.